smartsim/experiment.py

```diff
@@ -849,3 +860,14 @@ def _launch_summary(self, manifest: Manifest) -> None:
 
     def __str__(self) -> str:
         return self.name
+
+    def append_to_db_identifier_list(self, db_identifier: str) -> None:
+        """Check if db_identifier already exists when calling create_database"""
+        if db_identifier in self.db_identifiers:
+            logger.warning(
+                f"A database with the identifier {db_identifier} has already been made"
+                "An error will be raised if multiple databases are started"
```

Comment:
Missing white space at the end of strings, when concatenated the result will not be the desired one (i.e. "... madeAn ... startedwith").